systemd/systemd.go

```diff
@@ -62,21 +60,19 @@ func run(args ...string) ([]byte, error) {
 // systemctl. It's exported so it can be overridden by testing.
 var SystemctlCmd = run
 
-// jctl calls journalctl to get the JSON logs of the given services, wrapping the error if any.
-func jctl(svcs []string) ([]byte, error) {
-	cmd := []string{"journalctl", "-o", "json"}
-
-	for i := range svcs {
-		cmd = append(cmd, "-u", svcs[i])
+// jctl calls journalctl to get the JSON logs of the given services.
+func jctl(svcs []string, n string, follow bool) (io.ReadCloser, error) {
+	args := make([]string, 0, 2*len(svcs)+4)
```

Comment:
Ah, ok, I'm glad I've asked. It's not critical, but a comment would be nice indeed.